pkg/inclusion/get_commit.go

```diff
@@ -4,9 +4,26 @@ import (
 	"errors"
 
 	"github.com/celestiaorg/celestia-app/pkg/da"
+	"github.com/celestiaorg/celestia-app/pkg/shares"
 	"github.com/tendermint/tendermint/crypto/merkle"
 )
 
+func GetMultiCommit(cacher *EDSSubTreeRootCacher, dah da.DataAvailabilityHeader, startIndexes, lengths []uint32) ([]byte, error) {
+	shareCounts := make([]int, len(lengths))
+	for i := range lengths {
+		shareCounts[i] = shares.SparseSharesNeeded(uint32(lengths[i]))
+	}
+	commitments := make([][]byte, len(startIndexes))
+	for i := range startIndexes {
+		commitment, err := GetCommit(cacher, dah, int(startIndexes[i]), shareCounts[i])
+		if err != nil {
+			return nil, err
+		}
+		commitments[i] = commitment
+	}
+	return merkle.HashFromByteSlices(commitments), nil
```

Comment:
Q: Is the merkle hash of a single value just the sha256 hash?

Reply:
yeah, until we have multiple blobs, or for txs with a single blob, we're effectively taking the hash twice